torch/utils/collect_env.py

```diff
@@ -283,7 +283,7 @@ def get_pip_packages(run_lambda):
     # People generally have `pip` as `pip` or `pip3`
     # But here it is incoved as `python -mpip`
     def run_with_pip(pip):
-        out = run_and_read_all(run_lambda, f"{pip} list --format=freeze")
+        out = run_and_read_all(run_lambda, "{} list --format=freeze".format(pip))
```

Comment:
Maybe add comment that we specifically want older Python compatibility here so this is not updated accidentally.